contracts/tasks/miscs/get-vote-gauge-weight.ts

```diff
@@ -0,0 +1,284 @@
+import { task } from 'hardhat/config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { request, gql } from 'graphql-request'
+import { JsonRpcProvider } from '@ethersproject/providers'
+import BigNumberJs from 'bignumber.js'
+import { BigNumber } from 'ethers'
+
+// Constants
+const HOUR = 60 * 60 // in minute
+const DAY = HOUR * 24
+const WEEK = DAY * 7
+
+const URL = 'https://hub.snapshot.org/graphql'
+const SPACE = 'muuu.eth'
+
+// - Parameter
+const BORDER = {
+  top: 50,
+  bottom: 5,
```

Comment:
maybe 1?

Reply:
Sorry, It was the value used for verification.